src/redux/Dispatch/actions.js

```diff
@@ -309,3 +325,34 @@ export function updateTask(action, task) {
     }
   };
 }
+
+export function updateTaskList(action, taskList) {
+  return function (dispatch, getState) {
+    let date = selectSelectedDate(getState());
+
+    if (isSameDateTaskList(taskList, date)) {
+      switch (action) {
+        case 'v2:task_list:updated':
```

Comment:
I dont think it is necessary to have another `case` here as you have it already here https://github.com/coopcycle/coopcycle-app/pull/1968/files#diff-24dad85d996b9bee06ecd189b21c03f6fc11b00f28794cc7fcdc84942ae3eeeeR30